content/en/docs/chart_template_guide/looup_function.md

```diff
@@ -0,0 +1,50 @@
+---
+title: "Lookup Function"
+description: "Looking values from resources in a running cluster"
+weight: 17
+---
+
+The `lookup` can be used to reources from resources in a running cluster.
+The sinopsys of the lookup function is:
+`lookup apiVersion(string, not-empty), kind (string, not-empty), namespace(string, can be empty), name(string, can be empty) -> resource map[string]interface{}`
```

Comment:
```
lookup apiVersion(string, not-empty), kind (string, not-empty), namespace(string, can be empty), name(string, can be empty) -> resource map[string]interface{}
```

The format of this confused me at first and I know how the lookup function works. Could the type information be removed or moved and explained a little differently.

Reply:
improved, see if it's better.